app/controllers/concerns/sign_in/authentication.rb

```diff
@@ -23,6 +23,13 @@ def authenticate
       handle_authenticate_error(e)
     end
 
+    def load_user
+      @access_token = authenticate_access_token
+      @current_user = load_user_object
+    rescue SignIn::Errors::StandardError
+      nil
```

Comment:
Can we add a log here (just a normal rails log, probably not even a sentry log) that brings out the error? (Something along the lines of 'load_user not authenticated, reason=#{error}') 9/10 times it'll say 'Malformed JWT or whatever' but will prob be good just to have a marker in case there are other reasons why they aren't authenticating through this